### PR TITLE
Add more Syndication tests

### PR DESCRIPTION
**Pull request by hughbe**
RssFeed20Formatter was missing tests

### CODE REVIEW

File: src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs [hunk 1]
```diff
@@ -245,8 +245,7 @@ public SyndicationTextInput TextInput
         private SyndicationLink TryReadDocumentationFromExtension(SyndicationElementExtensionCollection elementExtensions)
         {
             SyndicationElementExtension documentationElement = elementExtensions
-                                      .Where(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace)
-                                      .FirstOrDefault();
+                                      .FirstOrDefault(e => e.OuterName == Rss20Constants.DocumentationTag && e.OuterNamespace == Rss20Constants.Rss20Namespace);
```
Comment (stephentoub) [inline src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs]: These aren't tests, they're product src changes.

Comment (hughbe) [reply src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs]: Reverted and moved to a different PR

File: src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs [hunk 2]
```diff
@@ -0,0 +1,2029 @@
...
+            <a10:updated>0001-01-01T00:00:00Z</a10:updated>
+        </item>
+        <item xml:base=""http://microsoft/relative"" item_name1="""" d3p1:item_name2="""" d3p1:item_name3=""item_value"" d2p2:item_name4="""" xmlns:d3p1=""item_namespace"">
+            <guid isPermaLink=""false"">id</guid>
+            <a10:author />
+            <a10:author author_name1="""" d4p1:author_name2="""" d4p1:author_name3=""author_value"" d2p2:author_name4="""" xmlns:d4p1=""author_namespace"">
+                <a10:name>author_name</a10:name>
+                <a10:uri>author_uri</a10:uri>
+                <a10:email>author_email</a10:email>
+                <Rss20FeedFormatterTests.Extension
```
Comment (stephentoub) [inline src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs]: Commented out?

Comment (hughbe) [reply src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs]: Slightly confused, what do you mean? I don't think anything is commented out in this test file

Comment (stephentoub) [reply src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs]: L1641 begins with //

**Comment by karelz** [comment]
@huanwu @Lxiamail can you please check it out and merge? It got approval from @stephentoub 2.5 weeks ago ... (sorry for late tagging)

File: src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs (hunk 2 above)
Comment (yujayee) [reply src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs]: @hughbe Is this line intended to be commented out?

**Comment by yujayee** [comment]
I'll go ahead and merge it once got clarification on the small comment question.

**Pull request merged**